learning-objectives/intl/es.yml

```
@@ -2087,3 +2087,286 @@ design-systems/atomic-design:
 design-systems/design-system:
   title: Utilizar sistemas de diseño
   description: Utilizar un sistema de diseño existente como base para la propuesta de diseño
+
+csharp:
+  title: C#
+
+csharp/data-types:
+  title: Tipos de datos
+
+csharp/data-types/primitives:
+  title: Tipos de datos primitivos
+  links:
+    - title: Tipos integrados (referencia de C#) - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/language-reference/builtin-types/built-in-types
+    - title: C# | Data Types (en inglés)
+      url: https://www.geeksforgeeks.org/c-sharp-data-types/
+
+csharp/data-types/non-primitives:
+  title: Tipos de datos no primitivos
+  links:
+    - title: Common Type System - Microsoft Docs
+      url: https://learn.microsoft.com/es-es/dotnet/csharp/fundamentals/types/#the-common-type-system
+
+  csharp/data-types/structs:
```

la indentacion es mal
```suggestion
csharp/data-types/structs:
```

---

Creo que no es necesario primitive vs non primitive, hay bastante documentación de tipos de datos.

Corregi la indentación, gracias!